faker.js types are not full

Stubs for https://github.com/Marak/faker.js are not complete.

When trying to call `faker.seed(1234)` there's a `flow` error on this method:

```
  10:13  error  Cannot call `faker.seed` because property `seed` is missing in object type (see https://github.com/facebook/flow/blob/v0.74.0/lib/faker_v4.x.x.js#L5)  flowtype-errors/show-errors
```

But, the method is there: https://github.com/Marak/faker.js/blob/master/lib/index.js#L153-L157
It is just missing from the declaration file: https://github.com/flow-typed/flow-typed/blob/efe6fe387a53c0b69209a962734bf9ba924c98f2/definitions/npm/faker_v4.x.x/flow_v0.25.x-/faker_v4.x.x.js

The same goes for other prototype methods, like `setLocale`. https://github.com/Marak/faker.js/blob/master/lib/index.js#L149-L151

I will send a PR with a fix shortly.

--- a/definitions/npm/faker_v4.x.x/flow_v0.25.x-/faker_v4.x.x.js
+++ b/definitions/npm/faker_v4.x.x/flow_v0.25.x-/faker_v4.x.x.js
@@ -1,8 +1,8 @@
 declare module "faker" {
-  declare type SeedType = void | number | $ReadOnlyArray<number>;
+  declare type SeedType = number | $ReadOnlyArray<number>;
 
   declare module.exports: {
-    seedValue: SeedType,
+    seedValue: ?SeedType,
     seed: (SeedType) => void,
     setLocale: (string) => void,
 

--- a/definitions/npm/faker_v4.x.x/flow_v0.25.x-/faker_v4.x.x.js
+++ b/definitions/npm/faker_v4.x.x/flow_v0.25.x-/faker_v4.x.x.js
@@ -1,5 +1,11 @@
 declare module "faker" {
+  declare type SeedType = void | number | $ReadOnlyArray<number>;
+
   declare module.exports: {
+    seedValue: SeedType,
+    seed: (SeedType) => void,
+    setLocale: (string) => void,
+
     address: {
       zipCode: (localeFormat?: string) => string,
       city: (mustacheTemplate?: string) => string,

--- a/definitions/npm/faker_v4.x.x/flow_v0.25.x-/test_faker_v4.x.x.js
+++ b/definitions/npm/faker_v4.x.x/flow_v0.25.x-/test_faker_v4.x.x.js
@@ -4,7 +4,6 @@ import faker from "faker";
 describe("prototype", () => {
   faker.seed(12);
   faker.seed([1, 2, 3]);
-  faker.seed(undefined);
   faker.seedValue;
   faker.setLocale('ru');
 });

--- a/definitions/npm/faker_v4.x.x/flow_v0.25.x-/test_faker_v4.x.x.js
+++ b/definitions/npm/faker_v4.x.x/flow_v0.25.x-/test_faker_v4.x.x.js
@@ -1,6 +1,14 @@
 import { describe, it } from "flow-typed-test";
 import faker from "faker";
 
+describe("prototype", () => {
+  faker.seed(12);
+  faker.seed([1, 2, 3]);
+  faker.seed(undefined);
+  faker.seedValue;
+  faker.setLocale('ru');
+});
+
 describe("address", () => {
   faker.address.zipCode();
   faker.address.zipCode("localeFormat");

